src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -32,6 +32,13 @@ const BLOODLILY_CONSUMERS: ActionKey[] = [
 	'AFFLATUS_MISERY',
 ]
 
+// Leaving out CURE3 due to having it only warn instead of error
+const GCD_HEALS: ActionKey[] = [
```

Comment:
Is the regen on Medica2/Regen enough to not include them? I cant think of many situations where the regen effect would be enough to not call it out?

Reply:
I'm on board for Regen but I'm sort of torn on Medica2. I can see heal check phases being started with Medica2 first while lilies are up in order to begin regen first and then using lilies after. However, I'm sure this as a case where if someone's smart enough to order their heals like that then they're smart enough to understand that the tip is selective in its application.

Reply:
Had a talk with people in the discord dev channel and its an awkward one. If we keep it to Cure Cure II and Medica for now